presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/ElasticsearchServer.java

```diff
@@ -13,16 +13,42 @@
  */
 package io.prestosql.elasticsearch;
 
+import com.google.common.io.Files;
 import com.google.common.net.HostAndPort;
 import org.testcontainers.elasticsearch.ElasticsearchContainer;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Map;
+
+import static com.google.common.io.Files.createTempDir;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.testcontainers.utility.MountableFile.forHostPath;
+
 public class ElasticsearchServer
 {
     private final ElasticsearchContainer container;
 
-    public ElasticsearchServer(String version)
+    public ElasticsearchServer(String image, Map<String, String> configuration)
+            throws IOException
     {
-        container = new ElasticsearchContainer("docker.elastic.co/elasticsearch/elasticsearch-oss:" + version);
+        container = new ElasticsearchContainer(image);
+
+        if (!configuration.isEmpty()) {
+            File tempDir = createTempDir();
+            tempDir.deleteOnExit();
```

Comment:
will this be deleted on exit recursively?

Reply:
It will not. I'll change the strategy (btw, we have the same problem in CassandraServer)